pxtlib/service.ts

```diff
@@ -98,6 +98,14 @@ namespace ts.pxtc {
         value: number;
     }
 
+    export type CodeLang = "py" | "blocks" | "ts"
+    export type SourceInterval = {
```

Comment:
it's confusing to use the same typename in different places

Reply:
ty, renamed for clarification